ingestr/src/utility.py

```diff
@@ -0,0 +1,18 @@
+import requests
```

Comment:
using generic names like `utils` or `utility` is considered bad practice for packages. 

Rather, we should name it something more specific. `http_client` for instance is a much better name.

Reply:
if anything, I would move these functions into the other file, keep them simple

Reply:
I agree with the naming since this will be used across different sources